docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
```

Comment:
Thunderdome is agile scrum practices including Story pointing (battles), Sprint Retrospectives, Story mapping, Async Daily Standup (team standup).